modules/kernel/src/main/scala-2.10/notebook/ReplCalculator.scala

```diff
@@ -230,7 +230,7 @@ class ReplCalculator(
           log.info(s"Killing job Group $jobGroupId")
           val thisSender = sender()
           repl.evaluate(
-            s"""sparkContext.cancelJobGroup("${jobGroupId}")""",
+            s"""globalScope.sparkContext.cancelJobGroup("${jobGroupId}")""",
```

Comment:
yes, serializable crap in 2.11 I think.

Indeed we can put things in different PRs too...